plugins/notifications-backend/src/service/router.ts

```diff
@@ -12,13 +12,22 @@ import {
   getNotifications,
   getNotificationsCount,
 } from './handlers';
+import { NotificationsQuerySorting } from './types';
 
 interface RouterOptions {
   logger: Logger;
   dbConfig: Config;
   catalogClient: CatalogClient;
 }
 
+const FieldsAllowedToOrderBy: NotificationsQuerySorting['fieldName'][] = [
```

Comment:
this should be in handler and not router. router is just one way to receive input. but the handler is responsible for validating it.